docs/bokeh/source/_static/switcher.json

```diff
@@ -1,7 +1,9 @@
 [
   {
     "name": "latest",
-    "url": "https://docs.bokeh.org/en/latest/"
+    "version": "3.4.1",
+    "url": "https://docs.bokeh.org/en/latest/",
+    "preferred": true
   },
   {
     "version": "3.4.0",
```

Comment:
FYI we could also remove this entry. It should have been removed on the last release, we only show the latest point release from each release level, but I forgot to edit it out. But we can try to remember to do it in the next release, too.